src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
```

Comment:
2. Exception handling:
   - It's advisable to handle exceptions in a structured way in a production application. You can consider adding a `@ExceptionHandler` method in a `@ControllerAdvice` class to handle exceptions globally.